pom.xml

```diff
@@ -58,6 +58,12 @@
       <version>2.0.0</version>
     </dependency>
 
+    <dependency>
+      <groupId>edu.tamu.weaver</groupId>
+      <artifactId>messaging</artifactId>
+      <version>2.0.0</version>
```

Comment:
This should reference the messaging module snapshot. The CI will pass when that snapshot is deployed to our maven repository.

Reply:
https://github.com/TAMULib/Cap/commit/5b578ee2425db316033b8108870f9ca77e4775a8